### ISSUE TITLE
Suggestion: `grabEntitiesFromRepository()` (Doctrine2 Module) should accept array parameter $order

### ISSUE THREAD

**Issue by ThomasLandauer** [Codeception/Codeception#5075]
Like Doctrine's `findBy()` function, `grabEntitiesFromRepository()` should accept an optional array as third parameter to add `ORDER BY` to the SQL query:
```php
grabEntitiesFromRepository('...', array(...), array('name' => 'DESC', 'id' => 'ASC');
```

**Comment by DavertMik** [comment]
Please make PR for that

**Comment by alexkunin** [comment]
If PR #5617 is accepted, it will be possible to get this functionality via criteria. See mentioned PR for details (code snippet in docs).

**Comment by Naktibalda** [comment]
Closed by #5617
